Source/Core/DolphinWX/LogWindow.cpp

```diff
@@ -359,5 +361,13 @@ void CLogWindow::Log(LogTypes::LOG_LEVELS level, const char* text)
   if (msgQueue.size() >= MSGQUEUE_MAX_SIZE)
     msgQueue.pop();
 
-  msgQueue.emplace(static_cast<u8>(level), StrToWxStr(text));
+  // Convert from UTF-8
+  wxString encoded_message = StrToWxStr(text);
+  // Fallback to OS locale if conversion failed
+  if (encoded_message.empty())
```

Comment:
That's true that empty messages are skipped but text isn't a string but a const char*. Should I really add ```text && *text```?